src/components/OverlayWindow/OverlayWindowView.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * TODO - Allow styles to be passed (eg: `classes` prop and className)
```

Comment:
@JoonLeeNIH Is it possible to pass Styles as a prop currently?

Reply:
No